### ISSUE TITLE
Intersting work! I have few code questions about.

### ISSUE THREAD

**Issue by RorschachChen** [HongHanh2104/T2A-Think-Twice-Before-Adaptation#2]
1. In T2AAdapter, self.entropy_fn is missing, so I defined:
self.entropy_fn = lambda x: -torch.sum(F.softmax(x, dim=1) * F.log_softmax(x, dim=1), dim=1)

2. self.e_margin is not defined, I set to 0.5. Is that ok?

3. self.filter_grad is not defined in both config file and class function.

4. in function compute_noise_tolerant_negative_loss(), 
when noise_type == "bernoulli":
flip_probs = 1 - probs
flipped_labels = torch.bernoulli(flip_probs).long()
flipped_labels shape is [N, 2]
according to the equation 10, I think the following should be executed first, then the normalizedloss
flipped_labels = flipped_labels.argmax(dim=-1)

Looking for your reply, thanks a lot. Have a good day~
